apps/showcase/src/app/app.component.ts

```diff
@@ -43,14 +45,19 @@ export class AppComponent implements OnDestroy {
 
   private subscriptions = new Subscription();
 
-  constructor(router: Router, private offcanvasService: NgbOffcanvas) {
+  constructor(
+    router: Router,
+    public readonly configConsoleDevtools: ConfigurationDevtoolsConsoleService,
+    public readonly localizationConsoleDevtools: LocalizationDevtoolsConsoleService,
```

Comment:
are we injecting this to trigger the constructor of the service as a side effect ?
I don't really like that, I'd rather have an explicit function like `.activate()` or something

Reply:
Done in the `main.ts`
Should we do it automatically when run `ng-add` of CMS packages ?